build/alpine/Dockerfile.prebuilder

```diff
@@ -95,3 +95,9 @@ RUN cd /tmp/ghc && \
     make install &> /dev/null && \
     mv /tmp/config.yaml /root/.stack/ && \
     rm -rf /tmp/ghc
+
+# Upgrade stack (there is no binary for stack-1.7.1 on alpine, so this is the easiest way.)
+run stack update && \
```

Comment:
Is there a reason you do the stack upgrade in the pre-builder, rather than the builder image? Can you use uppercase `RUN` for consistency?

Reply:
we're building builder more often, so it saves time to do it here, rather than once a week.  (earlier i thought it's once a ci run, but the argument still holds, even though it is weaker.  i still don't see a strong reason to have the distinction to begin with: i just build a new prebuilder in 45 minutes.)

yes, i'll fix the casing, thanks.